test/unit/run_cmd_tests.rb

```diff
@@ -0,0 +1,46 @@
+require 'assert'
+require 'cap-util/run_cmd'
+
+class CapUtil::RunCmd
+
+  class UnitTests < Assert::Context
+    desc "CapUtil::RunCmd"
+    setup do
+      @fake_cap = CapUtil::FakeCap.new
+      @fake_cap.current_path = "/a/current/path"
+      @fake_cap.release_path = "/dat/release/path"
+
+      @util = CapUtil::RunCmd.new(@fake_cap, 'ls -la')
+    end
+    subject{ @util }
+
+    should have_imeths :run
+
+    should "build a rake task to run" do
```

Comment:
@kellyredding - I think you want "build a cmd to run" instead of rake task here.